### ISSUE TITLE
Should measurement tools make noise

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fourier-making-waves#169]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/701.
From Slack:
>Kathryn Woessner:keyboard:  11:24 AM
For Fourier, should picking up and dropping the measurement tools make noise?
Chris Malley  11:24 AM
No. But feel free to suggest that in an issue.
Kathryn Woessner:keyboard:  11:25 AM
I'm not sure it needs it. But the dev test said there were sounds for everything, so I wanted to check
Chris Malley  11:26 AM
You’re right - I hadn’t considered the measurement tools in the notes.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Fourier: Making Waves‬
URL: https://phet-dev.colorado.edu/html/fourier-making-waves/1.0.0-dev.47/phet/fourier-making-waves_en_phet.html
Version: 1.0.0-dev.47 2021-08-29 19:29:57 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by arouinfar** [comment]
> For Fourier, should picking up and dropping the measurement tools make noise?

I think @KatieWoe is referring to the sound when an object is grabbed. For example, the book in Friction and the ruler in Gravity Force Lab must be grabbed before they can be moved. In both cases, there is a sound when these objects are grabbed, but there is not a sound specific to moving them around.

There is no grab action required for the measurement tools in Fourier. The grab action generally leads to a better UX for screen reader users, but this sim doesn't support interactive description. See https://github.com/phetsims/fourier-making-waves/issues/53#issuecomment-844311670 for more discussion about the grab action in other sims. 

The tools in this sim are movable as soon as they have focus, so I don't think there is a need for a sound here.

I think we can close this issue. Alternatively, we can defer it until it comes time to work on interactive description. We may need to add a grab action for the tools, and if we do, it should have a sound. I'll let @pixelzoom decide whether to close or defer.

**Comment by pixelzoom** [comment]
I'll close, since I think a future phase of a11y design should look at the sim holistically. And that's likely to be a ways put in the future.
